examples/src/forms__AirForm__render.py

```diff
@@ -0,0 +1,79 @@
+"""Example: Using AirForm.render to generate HTML for a simple contact form.
```

Comment:
This docstring at the beginning of the example is unnecessary and breaks the docstring inside of `forms.AirForm.render`.